docs/getting_started/Datus_tutorial.md

```
@@ -0,0 +1,257 @@
+# Datus Tutorial: A Complete Walkthrough for contextual data engineering
+*A step-by-step guide to understanding and practicing Contextual Data Engineering*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)